src/pages/mesh/advanced/headers.md

```diff
@@ -171,6 +165,21 @@ Including metadata prefixes the returned response headers with their source name
 }
 ```
 
+### Response tracking and debugging
+
+When reflecting headers, the following headers are included in the response to assist with tracking and debugging:
+
+- `cf-ray` - Generated automatically
+- `x-request-id` - Generated automatically, if the header is not provided in the request
+
+These headers allow you to track and debug requests by assigning a request-specific ID numbers. If you provide your own `x-request-id` in the request header, then the response includes the specified header instead of an automatically generated one. For example:
+
```

Comment:
```suggestion
These headers allow you to track and debug requests by assigning a request-specific ID numbers. If you provide your own `x-request-id` in the request header, then the sub-requests to the source will include the specified header instead of an automatically generated one. For example:

```